game-using-jsDOM/script.js

```diff
@@ -0,0 +1,219 @@
+// Array of words that we will use for the game 
+const words = ['penguins', 'elephant', 'koala', 'synonym', 'peanut',
+    'english', 'ruby', 'react', 'friday', 'airplane', 'banana', 'birthday',
+    'waterfall', 'trampoline', 'window', 'president', 'basketball', 'hiccup',
+    'cheerleader', 'kangaroo', 'groundhog', 'soccer', 'handball', 'caterpillar',
+    'shoelaces', 'hungry', 'fishing', 'monster'];
+
+/**
+ * Gets a random word from the words array declared on line 2
+ * @returns a word that has been randomly selected from the words_array
+ */
+function getWord() {
+    return (words[Math.floor(Math.random() * words.length)]);
```

Comment:
Would you mind adding a parameter in order to pass word list instead of reading from a global variable? We should keep our functions as "pure" as possible. Ref: https://www.google.com/amp/s/www.freecodecamp.org/news/what-is-a-pure-function-in-javascript-acb887375dfe/amp/

The code should looks like:

```ts
function getWord(wordList) { 
  return wordList[Math.floor(Math.random() * (wordList.length - 1))];
}
```

> Note:  Try not to forget that positions goes from 0 to length of array - 1, using just length of array could lead us to some issues

Reply:
Thank you! I totally forgot that it's the length-1. I have updated my code & have also changed getWord() to accept an array now. :)